frontend/src/components/eMIB/ActionViewEmail.jsx

```diff
@@ -102,7 +108,7 @@ class ActionViewEmail extends Component {
 
   render() {
     const action = this.props.action;
-    const visibleToNames = this.generateEmailNameList(action.emailTo);
+    const visibleToNames = this.generateEmailNameList(action.emailTo, true);
     const visibleCcNames = this.generateEmailNameList(action.emailCc);
```

Comment:
with the above change, this line does not break? or is 'isToField' just undefined?